test/system/src/main/java/io/pravega/test/system/framework/DockerRemoteSequential.java

```diff
@@ -0,0 +1,162 @@
+    }
+
+    private boolean isTestRunning(String containerName) {
+        boolean value = false;
+        try {
+            //list the service with filter 'serviceName'
+            List<Container> containers = CLIENT.listContainers(DockerClient.ListContainersFilterParam.withStatusRunning());
+            for (int i = 0; i < containers.size(); i++) {
+                ContainerInfo info = CLIENT.inspectContainer(containers.get(i).id());
+                if (info.name().equals(containerName)) {
+                    value = true;
+                    break;
+                }
+            }
+        } catch (DockerException | InterruptedException e) {
+            log.error("unable to list docker services {}", e);
+        }
+        return value;
+    }
+
+    private String startTest(String containerName, String className, String methodName) {
+
+        try {
+            CLIENT.pull(IMAGE);
+
+            ContainerCreation containerCreation = CLIENT.createContainer(setContainerConfig(methodName, className), containerName);
+            assertFalse(containerCreation.id().toString().equals(null));
+
+              id = containerCreation.id();
+
+            final Path dockerDirectory = Paths.get(System.getProperty("user.dir"));
+            try {
+                CLIENT.copyToContainer(dockerDirectory, id, "/data");
+            } catch (IOException | DockerException | InterruptedException e) {
+                log.error("Exception while copying test jar to the container {}", e);
+            }
+
+            // Inspect container
+            final ContainerInfo info = CLIENT.inspectContainer(id);
+
+            // Start container
+            CLIENT.startContainer(id);
+
+        } catch (DockerException | InterruptedException e) {
+            log.error("exception in starting container {}", e);
+        }
+        return id;
+    }
+
+    private ContainerConfig setContainerConfig(String methodName, String className) {
+
+        Map<String, String> labels = new HashMap<>(1);
+        labels.put("testMethodName", methodName);
+        labels.put("testClassName", className);
+
+        HostConfig hostConfig = HostConfig.builder()
+                .portBindings(ImmutableMap.of( "2375/tcp", Arrays.asList( PortBinding.of( System.getProperty("masterIP"), "2375" ) ) ) ).networkMode("host").build();
+
+        ContainerConfig containerConfig = ContainerConfig.builder()
+                .hostConfig(hostConfig)
+                .image(IMAGE)
+                .user("root")
+                .workingDir("/data")
+                .cmd("sh", "-c", "java -DdockerExecLocal="+ Utils.isDockerLocalExecEnabled()+ " -DmasterIP="+System.getProperty("masterIP")+ " -DskipServiceInstallation=" + Utils.isSkipServiceInstallationEnabled() + " -cp /data/build/libs/test-collection.jar io.pravega.test.system.SingleJUnitTestRunner " +
+                        className + "#" + methodName + " > "+  className + "#" + methodName + "server.log 2>&1" + "; exit $?")
```

Comment:
The test log file name should include test name

Reply:
It includes . the file name is` className#methodNameserver.log`